app/api/signup/schemas.py

```diff
@@ -0,0 +1,21 @@
+# from datetime import datetime
+# from typing import Optional
+# from uuid import UUID
+
+from pydantic import BaseModel
+
+
+class Signup(BaseModel):
+    # id: Optional[UUID] = None
```

Comment:
Please *delete* - not comment out lines you do not use. Only leave comments if they're actual code comments